src/main/java/net/pms/network/DbIdResourceLocator.java

```diff
@@ -27,9 +27,13 @@
 
 	public enum DbidMediaType {
 
-		TYPE_FILES("FID$", "object.item"), TYPE_ALBUM("ALBUM$", "object.container.album.musicAlbum"), TYPE_PERSON("PERSON$",
-			"object.container.person.musicArtist"), TYPE_PLAYLIST("PLAYLIST$",
-				"object.container.playlistContainer"), TYPE_VIDEO("VIDEO$", "object.container.storageFolder");
+		TYPE_AUDIO("FID$", "object.item.audioItem"),
+		TYPE_FOLDER("FOLDER$", "object.container.storageFolder"),
```

Comment:
This is not implemented and there is a question if we need it. Folders are not stored in the database.

Reply:
Yes you're right. In the beginning, I was searching only for "FILES" type. But then the more concrete types were necessary and "FILES" became obsolete. Thanks for figuring out.

Reply:
It's a good point but it presents a problem with the search approach, since it is valid for renderers to search for folders, isn't it?

Reply:
It is valid to search for "object.container.storageFolder", but the current implementation ignores this `upnp:class` yet.

Since the `SearchRequestHandler` will not create an URL-Link of type `TYPE_FOLDER`, we should not actually have an issue here in this concrete java class (in discovering the DLNAResource which is not implemented).